components/compliance-service/compliance.go

```diff
@@ -262,7 +262,7 @@ func getNotificationsConnection(connectionFactory *secureconn.Factory,
 	timeoutCtx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
 	defer cancel()
 	conn, err := connectionFactory.DialContext(timeoutCtx, "notifications-service",
```

Comment:
If the notification-service is not running, will this wait until the 60-second timeout and return a connection `conn` without an error?

Reply:
i don't think so. I removed the WithBlock so it doesn't wait for anything